pkg/catalogv2/system/system.go

```diff
@@ -176,7 +179,7 @@ func (m *Manager) install(namespace, name string, values map[string]interface{})
 		Charts: []types.ChartUpgrade{
 			{
 				ChartName:   name,
-				Version:     chart.Version,
+				Version:     desiredVersion,
```

Comment:
I'm not sure this logic is right.  If min version is 1 and nothing is install and version 2 is available, version 2 should be installed.